app/controllers/sessions_controller.rb

```diff
@@ -1,8 +1,13 @@
 class SessionsController < ApplicationController
+  cattr_accessor :code
   def create
+    SessionsController.token_code(params[:code])
     party = Party.find_by(identifier: request.env["omniauth.params"]["url"]) if request.env["omniauth.params"]["url"]
     if user = User.find_by(spotify_id: spotify_params(request.env['omniauth.auth'])[:spotify_id])
       session[:user_id] = user.id
+      user.update(access_token: spotify_params(request.env['omniauth.auth'])[:access_token])
+      user.update(refresh_token: spotify_params(request.env['omniauth.auth'])[:refresh_token])
+      user.update(expires_at: spotify_params(request.env['omniauth.auth'])[:expires_at])
```

Comment:
Can we also change expires at to a datetime like we talked about?

Reply:
Removed the column from users table since we are getting new access token off of user.updated_at